torchtune/training/quantization.py

```diff
@@ -219,3 +227,60 @@ def swap_lora_linear_with_qat(
                 activation_qat_config,
                 weight_qat_config,
             )
+
+
+def convert_to_float8_training(
+    model: nn.Module,
+    fp8_recipe_name: Optional[str] = None,
+) -> nn.Module:
+    """
+    Prepare the model for float8 training by swapping all `nn.Linear` with `Float8Linear`.
+
+    Args:
+        model (nn.Module): The model to swap linear layers on
+        fp8_recipe_name (Optional[str]): name to identify one of the pre-made recipes,
+            one of "tensorwise", "rowwise", and "rowwise_with_gw_hp". If not specified,
```

Comment:
Noob float8 q: seems like tensorwise gives the best perf in your runs, is that typically the case?  I assume the rowwise scaling will have less degradation, though doesn't seem to show up in the eval numbers at all. (Mainly want to gain some understanding of what a sensible default is for folks to use.)

Reply:
My understanding is rowwise is typically a bit slower but more accurate. I think we don't see the "more accurate" part in the experiments because for this workload tensorwise is already preserving accuracies very well, but it's possible we'll need rowwise for other workloads still